README.md

```diff
@@ -1,6 +1,16 @@
 Metamath website scripts
 
-This repository has various scripts to set up a Metamath website.
+This repository has various scripts to set up or reconfigure a Metamath website.
+
+If you just want to change the configuration of the existing Metamath website,
+change this repo's "main" branch. Then log in with `ssh root@us.metamath.org`
```

Comment:
Is that useful for you only?
Maybe mention who is the current maintainer and that anyone who wants to help shall contact you for access?

Reply:
Mario too, and anyone else with the permissions.

You're right, I can mention who has those permissions.

Reply:
By the way, I am willing to serve as a conduit for people who want to make website changes. Make a PR here and/or a thread on the mailing list, and we can explore new functionality or changes to existing things.

Reply:
I tweaked the text further and I *think* I resolve this. I'm going to mark the conversation resolved, if it's not resolved or you see other problems, please re-raise an issue!